### DISCUSSION TITLE
Is the API rate limit for a given operation shared or independent among different spawned instances of an authorized application?

### DISCUSSION THREAD

**Discussion by mpenna** [amzn/selling-partner-api-models#3757] [General]
I have an application where eventually many client instances spawned with the same app credentials (client id, secret, etc.) but having each their own auth tokens created through the use of the same seller's refresh token could be requesting data concurrently from the same operation endpoint. In such a situation, will they all be considered in aggregate towards said operation's burst and rate limit specifications or will each instance be individually considered for such usage?

**Comment by stefnats** [comment]
The API Call Limits are alway per Merchant ID <-> Application ID pair.
So if you have multiple applications, they have their own API Call Limits for each merchant.
Also, if you have multiple merchants for your application, each have their own API Call Limits.
That also means, that for the same seller account the region (EU/NA/FE) shares the API Call Limits, because a seller account has the same Merchant ID in all countries of a region.
